packages/avatars/stories/examples/MenuUsage.tsx

```diff
@@ -0,0 +1,157 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useState } from 'react';
+import { Story } from '@storybook/react';
+import {
+  Dropdown,
+  Trigger,
+  Menu,
+  MediaItem,
+  MediaFigure,
+  MediaBody,
+  ItemMeta
+} from '@zendeskgarden/react-dropdowns';
+import { Grid, Row, Col } from '@zendeskgarden/react-grid';
+import { Avatar } from '@zendeskgarden/react-avatars';
+import { Button } from '@zendeskgarden/react-buttons';
+import { PALETTE } from '@zendeskgarden/react-theming';
+
+export const MenuUsage: Story = () => {
+  const [highlightedItem, setHighlightedItem] = useState<number | null>();
+
+  return (
+    <Grid>
+      <Row>
+        <Col>
```

Comment:
Would `<Col="center">` center the menu buttons? I think in the Buttons stories they might have [centered buttons](https://github.com/zendeskgarden/react-components/pull/889/files#diff-6110424735f7b4751a85edd0cc84b336R36) so it might be nice to the menu buttons centered too.

Reply:
Updated.